testing/bsg_cache/regression_non_blocking/exclusion.el

```diff
@@ -0,0 +1,81 @@
+//==================================================
```

Comment:
Was this file supposed to be included?

Reply:
yes, that's the exclusion list for coverage

Reply:
Was it autogenerated?

Reply:
yes

Reply:
I'm afraid I don't understand the purpose of the file, so these questions may seem weird.

Why did you need to include an auto-generated file? How is it used? Why can't it be generated before it's used?

Reply:
when we open up the coverage report, we need to waive some coverage holes that are valid exclusions (e.g. default case that's never hit). Instead of going through the report each time we open up the report, we save those exclusions in a file, which is loaded next time.

Reply:
Got it, so this isn't easy to regenerate? If it isn't then I'm fine with this